dcpy/connectors/edm/recipes.py

```diff
@@ -101,35 +100,29 @@ def fetch_sql(name: str, version: str, local_library_dir):
     return target_file_path
 
 
-def import_recipe_dataset(
-    recipe_name: str,
-    *,
-    version="latest",
-    local_library_dir=LIBRARY_DEFAULT_PATH,
-    import_table_name: Optional[str] = None,
-):
+def import_dataset(dataset: Dataset, *, local_library_dir=LIBRARY_DEFAULT_PATH):
     """Import a recipe to local data library folder and build engine."""
-    config = get_config(recipe_name, version)
+    config = get_config(dataset.name, dataset.version)
```

Comment:
`version` field is optional for a `Dataset` defaulting to `None`, while `get_config` expects a (non-optional) string and won't coerce `None` to `"latest"` but will return a weird path

Reply:
Good call... Will address that.

Reply:
@fvankrieken resolved [here](https://github.com/NYCPlanning/data-engineering/pull/232/commits/50fc9862f3c26ab06cc02fbe054d675a3a31e95f)